sinks_test.go

```diff
@@ -166,6 +172,60 @@ func TestSHA256Sum(t *testing.T) {
 	}
 }
 
+func TestSliceSink(t *testing.T) {
+	t.Parallel()
+	input := Echo("testdata/multiple_files/1.txt\ntestdata/multiple_files/2.txt\ntestdata/multiple_files/3.tar.zip\n")
+
+	want := []string{
+		"testdata/multiple_files/1.txt",
+		"testdata/multiple_files/2.txt",
+		"testdata/multiple_files/3.tar.zip",
+	}
+	got, err := input.Slice()
+	if err != nil {
+		t.Error(err)
+	}
+
+	if !cmp.Equal(got, want) {
```

Comment:
It's a small point, but I always try to write `cmp.Equal(want, got)`, because then they follow in the same order on the next line: `cmp.Diff(want, got)`. If you diff the results in the opposite order, they look wrong, so I use this order for both the comparison and the diff.